studio/components/interfaces/Billing/AddOns/ComputeSizeSelection.tsx

```diff
@@ -23,14 +22,6 @@ const ComputeSizeSelection: FC<Props> = ({
   onSelectOption,
 }) => {
   const { ref } = useParams()
-  const { asPath } = useRouter()
-  useEffect(() => {
-    const hash = asPath.split('#')[1]
-    if (hash !== undefined) {
-      window.location.hash = ''
-      window.location.hash = hash
-    }
-  }, [asPath])
```

Comment:
Removed this to add to a more central file [here](https://github.com/supabase/supabase/pull/14509/files#diff-a8c131a5c22f796169acad58da47ed1fbc47c41f3f33a2e1b90e0c84d5cc08e6R136)

Another side effect is that we have less duplicated code. Potentially more performant as well as useEffect only has to run once only